src/sass/landing_pages/usb.scss

```diff
@@ -0,0 +1,423 @@
+    background-size: contain;
+    cursor: pointer;
+    z-index: 9;
+}
+
+.grill-ico {
+    background: url('../images/usb_page/menu.svg') no-repeat;
+    background-position: center;
+}
+
+.navbar-fixed-top {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 1030;
+}
+
+.top-nav-collapse {
+    a {
+        color: #ffffff;
+    }
+    a:hover {
+        color: #cccccc;
+    }
+}
+
+.page-scroll, .logo-scroll, .noSelect, #push {
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-tap-highlight-color: transparent;
+}
+
+.logo-wrapper {
+    padding: 3px 0;
+    margin: 0;
+    margin-top: 0px;
+}
+
+.section-md {
+    padding: 30px 0;
+}
+
+.section-lg {
+    padding: 80px 0;
+}
+
+.section-title {
+    font-size: 36px;
+    font-weight: 300;
+    margin-top: 80px;
+    margin-bottom: 0px;
+}
+
+.coin-section {
+    border-top: 3px solid #e98024;
+    border-bottom: 3px solid #e98024;
+}
+
+.home--header {
+    width: 100%;
+    background: linear-gradient(rgba(42, 48, 82, 0.6), rgba(42, 48, 82, 0.1)), url('../images/usb_page/header-image_edited-sm.jpg') no-repeat;
+    background-size: cover;
+    padding: 50px;
+    background-position: center;
+    min-height: 75vh;
+    position: relative;
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    -ms-overflow-style: scrollbar;
+}
+
+.header-content {
+    text-align: left;
+    width: 1200px;
+    max-width: 100%;
+    padding: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+
+.header-title {
+    font-size: 48px;
+    line-height: 60px;
+    margin-bottom: 30px;
+}
+
+.header-sub {
+    display: block;
+    margin: 5px 0;
+    font-size: 36px;
+    font-weight: 300;
+}
+
+.earn-interest {
+    width: 800px;
+    max-width: 100%;
+}
+
+.usb-coin {
+    width: 150px;
+    max-width: 100%;
+}
+
+.separator-line-thin-gray-xs {
+    height: 1px;
+    background-color: #dedede;
+    margin: 0;
+}
+
+.separator-line-thin-gray {
+    height: 1px;
+    background-color: #dedede;
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+.separator {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    height: 1px;
+}
+
+.separator-md {
+    height: 20px;
+}
+
+.separator-lg {
+    height: 30px;
+}
+
+.separator-xl {
+    height: 50px;
+}
+
+.bottom-white-border {
+    border-bottom: 1px solid #ffffff;
+}
+
+.right-white-border {
+    border-right: 1px solid #ffffff;
+}
+
+.icon-xl {
+    display: inline-block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+    width: 150px;
+    max-width: 150%;
+    height: 150px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+}
+
+.icon-buy {
+    background: url('../images/usb_page/buy.svg') no-repeat;
+    background-position: center;
```

Comment:
how about using only `background-image: url('..');` for the icons? `background-repeat` and `background-position` property has been declared in line 200 and 201 above